src/server/pps/cmds/cmds.go

```diff
@@ -249,7 +299,7 @@ All jobs created by a pipeline will create commits in the pipeline's repo.
 		}),
 	}
 
-	var pipelinePath string
+	var createPipelinePath string
```

Comment:
I think it's fine to just reuse the variable as a argument for both create pipeline and update pipeline. We do that with a bunch of other command arguments and it works fine because only one of the commands is run per invocation of the program.